<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Storage</class>
 <widget class="QMainWindow" name="Storage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Storage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>710</width>
        <height>540</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Button_Dodaty">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Добавити</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Button_Delete">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Видалити</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Button_Spisaty">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Списати</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
